rasa/utils/schemas/stories.yml

```diff
@@ -36,7 +36,7 @@ mapping:
                 - type: "map"
                   mapping:
                     regex;(.*):
-                      type: "text"
+                      type: "any"
```

Comment:
@erohmensing This fixes the yaml problem and `true` without quotes works again. But I have no idea if this is the proper way of doing this.